pkg/porter/build.go

```diff
@@ -94,24 +94,20 @@ func (o *BuildOptions) parseCustomInputs() error {
 }
 
 func (p *Porter) Build(ctx context.Context, opts BuildOptions) error {
-	ctx, log := tracing.StartSpan(ctx)
-	defer log.EndSpan()
-
-	opts.Apply(p.Context)
```

Comment:
Do you mean to remove this line of code?

Reply:
Yes but it should have been in my follow-up PR. This snuck in too early! I'll revert this line and remove it in the other PR where I clean up where the --debug flag is defined.